charts/csm-encryption-rekey-controller/Chart.yaml

```diff
@@ -6,10 +6,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
+version: 0.3.0
```

Comment:
@xuluna same as above

Reply:
Same here, dellemc/csm-encryption-rekey-controller image is updated from 0.2 to 0.3